.github/workflows/update.yml

```diff
@@ -22,13 +22,15 @@ jobs:
           go-version-file: go.mod
       - run: go install golang.org/x/tools/cmd/goimports@latest
       - uses: arduino/setup-task@v2
-      - run: wget -O openapi.json https://api.aiven.io/doc/openapi.json
```

Comment:
🔎 No need to do this here, `generate` handles downloading the spec now: https://github.com/aiven/go-client-codegen/blob/b6808be0cdcfbece179e0a0d7c61c39bc26f0beb/Taskfile.yml#L21